data-collections-exercises.md

```diff
@@ -10,36 +10,50 @@ Access the second value in `planeteers`.
 
 ```rb
 # Your answer here
+ planeteers[1]
 ```
 
 Add "Heart" to the end of `planeteers`.
 
 ```rb
 # Your answer here
+ planeteers.push("Heart")
 ```
 
 Remove "Captain Planet" from the array (without using a method).
 
 ```rb
 # Your answer here
+new_planeteers =[]
+
+planeteers = planeteers.map do |i| 
+    next if i == "Captain Planet"
+    new_planeteers.push(i)
+end
+
+planeteers = new_planeteers
```

Comment:
this is a great way to remove Captain Planet but still, you used a method. There is a simple to solve this problem which is `planeteers - ["Captain Planet"]`